winsup/cygwin/mm/malloc_wrapper.cc

```diff
@@ -50,13 +50,26 @@ import_address (void *imp)
 {
   __try
     {
+#if defined(__aarch64__)
+      uint32_t opcode = *(uint32_t *) imp;
```

Comment:
This seem to be a one-purpose solution, here for the `malloc`, while the RC of the issue seem to be much more generic to all `SIGFE` functions. I'd expect the fix in `mkiport` or `gendef`.

Reply:
Proper implementation for SIGFE should not change this code. This code handles address resolving from the jump table for aarch64 architecture.